crtp/README.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Curiously Recurring Template Pattern
+language: en
+---

```

Comment:
check out the required yaml frontmatter at https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute